From a4c5e656e895522d140a419f58a5f8db69a931d1 Mon Sep 17 00:00:00 2001
From: Pankaj Dev <pankaj.dev@st.com>
Date: Mon, 18 Jul 2022 16:19:40 +0530
Subject: [PATCH] arm64: defconfig: Enable USB2Host and USB3DRD for stm32mp25

Enable USB2 host and DWC3 glue drivers for stm32mp25, and related
USB2 PHY used for both controllers.
Note: ComboPHY used for USB3 operations is already enabled.

Change-Id: Idda60c42f1678461e24bf3064dff5dad6dcf2b65
Signed-off-by: Pankaj Dev <pankaj.dev@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 3 +++
 1 file changed, 3 insertions(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -8,6 +8,7 @@ CONFIG_IIO_STM32_TIMER_TRIGGER=m
 CONFIG_MFD_STM32_LPTIMER=m
 CONFIG_MFD_STM32_TIMERS=m
 CONFIG_OF_OVERLAY=y
+CONFIG_PHY_STM32_USB2PHY=y
 CONFIG_PWM_STM32=m
 CONFIG_PWM_STM32_LP=m
 CONFIG_REGULATOR_ARM_SCMI=y
@@ -21,3 +22,5 @@ CONFIG_STM32_LPTIMER_CNT=m
 CONFIG_STM32_TIMER_CNT=m
 CONFIG_TEE_REMOTEPROC=y
 CONFIG_USB_HIDDEV=y
+CONFIG_USB_DWC3_STM32=y
+CONFIG_USB_STM32_USBH=y
